event/bus.go

```diff
@@ -60,6 +60,11 @@ func NewBus(callerMap *CallerMap) *Bus {
 	}
 }
 
+// SetCallerMap updates a bus to use a specific set of callers.
```

Comment:
Question: Is there a way that the fact that callermaps are recreated from scratch via scene loop be worked in here?